### DISCUSSION TITLE
desires to add to CLI/Utils in future

### DISCUSSION THREAD

**Discussion by TfedUD** [PH-Tools/honeybee_ph#3] [Ideas]
Hi, 
   Thought I'd start to list some of the nifty utils I've been wanting to build out for WP; since either here/PHX is where I'd be putting them if I ever have time to do em, and didn't want to forget. Some feel kinda CLI-ish; some very well don't belong in this project I think; but didn't know where should write down for now..
 

1.  Take variant from one file.wufi.XML and append to variants of another
2.  WP.mwp inputs to dataframe/spreadsheet:
      - possibly tabular by wp-tree-branch (seems like would jive)
      - i.e:
      ```python
         variant[1].internal_loads_table.head()
         #>> prints pandas df for table
      ```

**Comment by ed-p-may** [comment]
Definitely. I think a WP--->PHX--->HBJSON pathway is definitely going to be a useful thing to build out at some point. If you can get the data back from XML into PHX, could output format to any number of things (pd, etc.)

**Reply by TfedUD** [reply]
NICE!!
